geps/gep-1713/index.md

```diff
@@ -0,0 +1,380 @@
+# GEP-1713: Standard Mechanism to Merge Multiple Gateways
+
+* Issue: [#1713](/kubernetes-sigs/gateway-api/issues/1713)
+* Status: Provisional
+
+(See status definitions [here](overview.md#status).)
+
+## tl;dr
+
+The Gateway Resource is a contention point since it is the only place to attach listeners with certificates. We propose a mechanism to allow distinct Gateway resources to be logically merged.
+
+## Goals
+
+- Define a mechanic to merge multiple Gateways (logically)
+- Define a set of acceptable properties that can be merged and their semantics
```

Comment:
@dprotaso Overall the only properties I see defined here are based on Gateway parents and children.  Does this mean we don't consider whether listeners are "[compatible](https://github.com/kubernetes-sigs/gateway-api/blob/main/apis/v1/gateway_types.go#L159)" or "[distinct](https://github.com/kubernetes-sigs/gateway-api/blob/main/apis/v1/gateway_types.go#L95)" as described in the `GatewaySpec`?

Reply:
I believe this is sort of covered in the `listener precedence` sections and the `gateway conditions` section in the GEP.

I'll close this out end of week. Let me otherwise if those sections doesn't answer your question.

Reply:
Circling back this is covered under the section "Merging Spec Fields"

children can't  set `spec.address` and is limited to what they can set in `spec.infrastructure`

Reply:
Ok.  If you don't mind changing the word "mechanic" to "mechanism" at some point, I'll comment in those sections, and you can close this out.

Reply:
done